sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/pkversion/PartitionLoadBalancerImpl.java

```diff
@@ -119,30 +119,30 @@ private Mono<Void> run(CancellationToken cancellationToken) {
                         if (cancellationToken.isCancellationRequested()) return Mono.empty();
                         return this.partitionController.addOrUpdateLease(lease);
                     })
-                    .then(Mono.just(this)
+                    .then();
```

Comment:
one extra `then()`?

Reply:
lol yes will need it as we have two versions. So the same fix needs to apply in both versions